### DISCUSSION TITLE
Icon Accessibility Stuff

### DISCUSSION THREAD

**Discussion by justlimeguy** [godotengine/godot-proposals#8452] [Editor]
So, I was using different versions of godot and i found it confusing on how every i con looked the same.
it is not bad if using only one engine but when installing new version on the engine makes it confusing .
So i came up with this concept for the godot engine's editor for future releases.
i hope you will like it.
![icon-concept](https://github.com/godotengine/godot-proposals/assets/142280135/64b0a879-1010-447b-a111-47a267d6d19c)

basically have version name written somewhere to avoid any confusion, when working outside of editor.

**Comment by AThousandShips** [comment]
Thank you for your proposal, closing as a duplicate of:
* https://github.com/godotengine/godot-proposals/issues/541

Which discusses this in more general terms

**Reply by justlimeguy** [reply]
i was talking about just adding a version number, no an entirely different icon. Just to differentiate between 4.0, 3.x and 4.2

**Reply by AThousandShips** [reply]
Yes and that's touched on in that discussion

**Reply by Calinou** [reply]
Adding the version number to the icon probably won't happen for two reasons:

- It's very uncommon for apps to display their version number in the icon, so Godot would be straying from the expected convention.
- The version number would be hard to read at small sizes like 32×32 or worse, 16×16.

A dev/beta/RC/.NET badge doesn't have this issue because you'll largely rely on its color and/or position to identify it, rather than the text written on it.
